styles/variables-silicon.less

```diff
@@ -215,6 +215,14 @@
 @agate-incrementslider-vertical-button-increment-border-radius: @agate-incrementslider-horizontal-button-decrement-border-radius;
 @agate-incrementslider-vertical-button-increment-border-width: @agate-incrementslider-horizontal-button-decrement-border-width;
 
+// Indicator
+// ---------------------------------------
+@agate-indicator-align-self: center;
+@agate-indicator-border-radius: @agate-indicator-height / 2;
+@agate-indicator-display: flex;
+@agate-indicator-height: 9px;
+@agate-indicator-margin-right: 21px;
+
```

Comment:
Did this get copied from another PR? Or is this branch dependent on the other branch? #268

Reply:
`ThumbnailItem` inherits styles from `Item`. This branch is dependent on this branch: https://github.com/enactjs/agate/pull/281